<ui version="4.0" >
 <class>UpDownLoadDialog</class>
 <widget class="QDialog" name="UpDownLoadDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="updateLabel" >
     <property name="text" >
      <string>Updating ...</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="uploadNameLabel" >
     <property name="text" >
      <string>Filename</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="minimumSize" >
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="globalStateLabel" >
     <property name="text" >
      <string>Global progression ...</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="globalBar" >
     <property name="minimumSize" >
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="textDirection" >
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpDownLoadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpDownLoadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>273</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel" >
     <x>282</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
